app/views/ControlWindow.js

```diff
@@ -0,0 +1,168 @@
+/**
+ * @author Bingzheng Feng
+ * @module views/ControlWindow
+ * @requires models/Commands
+ * @requires util/Validator
+ * @requires util/Logger
+ * @requires models/TelemetryData
+ * @requires util/Template
+ * @requires electron
+ * @listens models/TelemetryData~TelemetryData:data_received
+ * @copyright Waterloo Aerial Robotics Group 2016
+ * @licence https://raw.githubusercontent.com/UWARG/WARG-Ground-Station/master/LICENSE
+ * @description Responsible for displaying control pitch, yaw, roll
+ */
+
+var remote = require('electron').remote;
+var Template = require('../util/Template');
+var TelemetryData = remote.require('./app/models/TelemetryData');
+var Logger = remote.require('./app/util/Logger');
+var Validator = require('../util/Validator');
+var Commands = remote.require('./app/models/Commands');
+var Channel_Conf = remote.require('./config/vehicle-config');
+var PicPilot_Conf = remote.require('./config/picpilot-config');
+
+module.exports = function (Marionette) {
+
+  return Marionette.ItemView.extend({
+    template: Template('ControlWindow'),
+    className: 'controlWinodw',
+
+    ui: {
+      front23: '.controlWindow.circle.front.twothree',
+      front14: '.controlWindow.circle.front.onefour',
+      ch1_text: '.controlWindow.value.one',
+      ch2_text: '.controlWindow.value.two',
+      ch3_text: '.controlWindow.value.three',
+      ch4_text: '.controlWindow.value.four',
+      autoPilot_btn: '.controlWindow.auto-pilot',
+    },
+
+    events: {
+    },
+
+    initialize: function () {
+      this.throttle = 0;
+      this.roll = 0;
+      this.pitch = 0;
+      this.yaw_rudder = 0;
+      this.auto_ON_OFF = true;
+      this.control_off = false;
+      this.type = PicPilot_Conf.get('type');
+      
+      this.aircraft_channel_callback = this.aircraftChannelCallback.bind(this);
+      TelemetryData.addListener('aircraft_channels', this.aircraft_channel_callback);
+    },
+    
+    aircraftChannelCallback: function(data){
+      var ch1 = data[Channel_Conf.get(this.type)['throttle']];
+      var ch2 = data[Channel_Conf.get(this.type)['roll']];
+      var ch3 = data[Channel_Conf.get(this.type)['pitch']];
+      var ch4 = data[Channel_Conf.get(this.type)['rudder_yaw']];
+      var ch5 = data[Channel_Conf.get(this.type)['autopilot_ON_OFF']];
+      // if RC controller is OFF
+      if(ch1 < -1024 || ch2 < -1024 || ch3 < -1024 || ch4 < -1024 || ch5 < -1024 || ch1 > 1024 || ch2 > 1024 || ch3 > 1024 || ch4 > 1024 || ch5 > 1024){
```

Comment:
1) Big if statement like this calls for a function use to clean up the logic
2) This isn't going to check if the rc controller is off. If you convert to a tick value when receiving PWM, you may actually get over 1024, and it'll still be valid. A valid check and more simple check would be to detect -10000 as the indicator, as its coded in the picpilot to do (-10000 means disconnect).  In addition, because youre code heavily relies on the fact that the values are going to be scaled (and not raw), I woulld add the show_scaled_pwm check to make sure that the inputs are what you are expecting